scripts/build-for-testing.sh

```diff
@@ -100,11 +87,11 @@ function xcb() {
 }
 
 # Run xcodebuild
-sudo xcode-select -s /Applications/Xcode_16.1.app/Contents/Developer
+sudo xcode-select -s /Applications/Xcode_15.3.app/Contents/Developer
 xcb "${flags[@]}"
 echo "$message"
 
 # Zip build-for-testing into MyTests.zip
-cd build-for-testing/${SCHEME}/Build/Products
```

Comment:
`Double quote to prevent globbing and word splitting.` ([SC2086](https://www.shellcheck.net/wiki/SC2086))